spec/druid_tools_spec.rb

```diff
@@ -6,6 +6,8 @@
   let(:tree1) { File.join(fixture_dir, 'cd/456/ef/7890/cd456ef7890') }
   let(:strictly_valid_druid_str) { 'druid:cd456gh1234' }
   let(:tree2) { File.join(fixture_dir, 'cd/456/gh/1234/cd456gh1234') }
+  let(:access_druid_str) { 'druid:cd456ef0987' }
```

Comment:
could the last 4 digits be really different, like 9999?

Reply:
yup! done.